app/customer.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+
+from app.car import Car
+
+
+class Customer:
+    picked_shop = None
+    costs_travel = 0
+
+    def __init__(self, name: str, money: int) -> None:
+        self.name = name
+        self.money = money
+
+    def money_sum(self) -> None:
+        print(f"{self.name} has {self.money} dollars")
+
+    def cost_trip(self, car: Car, product_cart: dict, shops: list) -> bool:
+        min_costs_shop = {}
+
+        for shop in shops:
+            distance = car.calculate_distance(shop["location"])
+            sum_cart = {key: product_cart[key] * shop["products"][key]
+                        for key in product_cart}
+            cost_travel = (sum(sum_cart.values()) + car.road_cost(distance))
+            print(f"{self.name}'s trip to the "
+                  f"{shop['name']} costs {cost_travel}")
+            min_costs_shop.update({shop["name"]: cost_travel})
+
+        min_costs_shop_sorted = dict(sorted(min_costs_shop.items(),
+                                            key=lambda value: value[1]))
+
+        name_shop, costs = next(iter(min_costs_shop_sorted.items()))
+
+        self.picked_shop = name_shop
+        self.costs_travel = costs
+        if costs > self.money:
+            print(f"{self.name} doesn't have enough "
+                  f"money to make a purchase in any shop")
+            return False
+
+        if costs <= self.money:
+            print(f"{self.name} rides to {name_shop}")
+            date_time_obj = datetime(2021, 4, 1, 12, 33, 41)
```

Comment:
```suggestion
            date_time_obj = datetime.now()
```

Reply:
When I put datetime.now() I get an error in the tests that it should be 'Date: 04/01/2021 12:33:41\n' but not the current date. "Date: 01/01/2024 16:42:10\n".